client/.env.sample

```diff
@@ -1,2 +1,2 @@
-REACT_APP_LOCAL=AUTH0_CALLBACK_URL
-REACT_APP_API=BASE_SERVER_URL
\ No newline at end of file
+REACT_APP_LOCAL=http://localhost:3000/auth
+REACT_APP_API=http://localhost:5000
```

Comment:
The `.env.sample` file is meant to hold dummy data so that future devs know what a `.env` file should look like, approximately. 

You should definitely revert this change instead of hardcoding it. 🙏